pkg/ebpf/tracer.go

```diff
@@ -92,6 +92,22 @@ func NewTracer(config *Config) (*Tracer, error) {
 		return nil, fmt.Errorf("could not load bpf module: %s", err)
 	}
 
+	// Enable kernel probes used for offset guessing.
+	// TODO: Disable them once offsets have been figured out.
+	for _, probeName := range offsetGuessProbes(config) {
+		if err := m.EnableKprobe(string(probeName), maxActive); err != nil {
+			return nil, fmt.Errorf(
+				"could not enable kprobe(%s) used for offset guessing: %s",
+				probeName,
+				err,
+			)
+		}
+	}
+
+	if err := guess(m, config); err != nil {
```

Comment:
nit: can we rename this to `guessOffsets`

Reply:
👍